src/pe_reports/data/data_schema.sql

```diff
@@ -144,6 +138,7 @@ CREATE TABLE IF NOT EXISTS public.hibp_exposed_credentials
     root_domain text,
```

Comment:
Line above this org, should be changed to organization_uid, and a FK should be added